gitlab/datadog_checks/gitlab/data/conf.yaml.example

```diff
@@ -153,9 +153,9 @@ instances:
   - gitlab_url: <GITLAB_URL>
 
     ## @param prometheus_endpoint - string - required
-    ## URL of the metrics endpoint of prometheus
+    ## URL of the Prometheus metrics endpoint of gitlab
     #
-    prometheus_endpoint: http://<PROMETHEUS_ENDPOINT>:<PROMETHEUS_PORT>/metrics
+    prometheus_endpoint: http://<GITLAB_URL>/-/metrics
```

Comment:
Currently you can only either collect the new metrics or the old "legacy metrics" since the urls for them are different "/-/metrics" vs "/metrics". I'm not sure of the best way to incorporate both using `OpenMetricsBaseCheck`

Reply:
I think it's fine to document the `/-/metrics` endpoint by default since it's what Gitlab officially [documents](https://docs.gitlab.com/ee/administration/monitoring/prometheus/gitlab_metrics.html#collecting-the-metrics).

Does the legacy endpoint still work btw?

Reply:
It does work but it just emits prometheus-specific metrics which aren't specific to gitlab.

Reply:
What would happen if a customer upgraded an agent with an existing check, but these changes then get introduced?  They would conceivably have the same (older) config.  Would this check still run successfully?

Reply:
Yes that's the idea! I left in the old e2e test so we know it still works for old config. In https://github.com/DataDog/integrations-core/pull/5971/files#diff-4c067887ce93cf3e795b69176c4cee4dR80 , I check for both `prometheus_url` and `prometheus_endpoint`. The one thing we need to investigate is if we can modify metric type submission for the new one and not affect the older metrics/dashboards/monitors.